### DISCUSSION TITLE
About Zipping JS Files

### DISCUSSION THREAD

**Discussion by Minemetero** [Minemetero/Table-Tennis-Counter#13] [Ideas]
关于压缩网页JS文件

**Comment by lingbopro** [comment]
emmm

这个嘛
用Actions挺好的
具体怎么弄还得思考一下啊

**Comment by lingbopro** [comment]
找到了一篇文章：<https://blog.csdn.net/weihaifeng163/article/details/84959354>  
这个讲的还算比较详细  
回头可以试试

**Comment by lingbopro** [comment]
还有一篇 [文章](https://juejin.cn/post/6844903933370957831)  

我个人觉得 Babel 比较方便，可以顺带处理兼容性  
Terser 的特点就是压缩出来的比较小一些  
UglifyJS 感觉没啥特点，跟 Terser 差不多，但是不支持ES6  

我还是比较推荐 Babel 的

**Reply by Minemetero** [reply]
压缩一下加载网页应该会更快

**Reply by lingbopro** [reply]
> 压缩一下加载网页应该会更快

那肯定啊
